src/extension.ts

```diff
@@ -731,15 +780,20 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
   vscode.workspace.workspaceFolders?.map((workspaceFolder) => {
     const uri = workspaceFolder.uri;
     const { configName } = connectionTarget(uri);
-    const serverName = notIsfs(uri) ? config("conn", configName).server : configName;
+    const conn = config("conn", configName);
```

Comment:
Can we use the standard `vscode.workspace.getConfiguration()` API instead of this old custom one? It would be nice to phase that out in the future.

Reply:
At this stage I'd prefer not to make unnecessary changes. The `config(...)` function does a number of things that mean I don't think `vscode.workspace.getConfiguration()` is a drop-in replacement.